index.html

```diff
@@ -4,9 +4,6 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
-    <link rel="stylesheet" href="styles.css">
-</head> 
```

Comment:
- [ ] Great job so far @NgachaIan I see you have implemented the previous reviewer's suggestion. However, it looks like you deleted some parts of the `head` section of your code. Currently, your code is missing the link to the `stylesheet`, the `title`, and the closing `head` tag. 
- [ ] As a result of this, I was unable to check if the styles adjustments the previous reviewer suggested in [this comment](https://github.com/NgachaIan/Personal_Portfolio/pull/6#discussion_r1228222823) were implemented. So kindly fix this :100:  
- Your Current UI

[Screencast from 06-17-2023 02:40:53 PM.webm](https://github.com/NgachaIan/Personal_Portfolio/assets/102699958/d2583489-67d0-485a-977c-85f865b745dd)